src/data_type/document.details.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const VALIDATION_PATTERN = /^([A-Za-z_]*) ([A-Za-z]{3}) ([A-Za-z0-9]{1}).*$/;
+const TYPE_INDEX = 0;
+const COUNTRY_INDEX = 1;
+const NUMBER_INDEX = 2;
+const EXPIRATION_INDEX = 3;
```

Comment:
Is there a concept of _private_ in PHP?
Should these be _private_?

Reply:
You mean in Node.js?
Not really, unless you use typescript.